README.md

```diff
@@ -53,7 +58,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+The second line of the code uses = and by doing that it assigns the value of sunny to the word weather.  So anytime you input weather into the REPL it will output sunny.  The third line is asking us to return a boolean value (true or false).  Since the phrase weather has been assigned a value of "sunny", that should return a true statement.
```

Comment:
Sort of. The third line isn't _asking_us anything, exactly. It's a comparison operation; it compares the values of `weather` and "sunny", and evaluates to `true` if the two are equal.